iina/VideoTime.swift

```diff
@@ -49,30 +49,22 @@ class VideoTime {
   }
 
   convenience init?(_ format: String) {
-    let split = format.split(separator: ":").map { (seq) -> Int? in
-      return Int(String(seq))
-    }
-    if !(split.contains {$0 == nil}) {
-      // if no nil in array
-      if split.count == 2 {
-        self.init(0, split[0]!, split[1]!)
-      } else if split.count == 3 {
-        self.init(split[0]!, split[1]!, split[2]!)
-      } else {
-        return nil
-      }
-    } else {
-      return nil
-    }
+    let split = Array(format.split(separator: ":").map { String($0) }.reversed())
```

Comment:
I don’t do a lot of swift coding so I’m not sure there isn’t a better way to do this.

Reply:
I think you should be ok with just `let split = format.split(separator: ":").reversed()`?